esmvaltool/recipes/recipe_kcs.yml

```diff
@@ -0,0 +1,153 @@
+  - {dataset: CCSM4, project: CMIP5, mip: Amon, exp: [historical, rcp60], ensemble: "r(1:4)i1p1", start_year: 1961, end_year: 2099}
+  - {dataset: CCSM4, project: CMIP5, mip: Amon, exp: [historical, rcp85], ensemble: "r(1:4)i1p1", start_year: 1961, end_year: 2099}
+
+
+preprocessors:
+  preprocessor_global:
+    custom_order: true
+    area_statistics:
+      operator: mean
+    annual_statistics:
+      operator: mean
+    anomalies:
+      period: full
+      reference:
+        start_year: 1981
+        start_month: 1
+        start_day: 1
+        end_year: 2010
+        end_month: 12
+        end_day: 31
+      standardize: false
+    multi_model_statistics:
+      span: full
+      statistics: [p10, p90]
+  preprocessor_local: &extract_NL
+    extract_point:
+      longitude: 6.25
+      latitude: 51.21
+      scheme: linear
+
+
+diagnostics:
+  global_matching:
+    description: >
+      - Make a plot of the global mean temperature change according to all datasets (defined above)
+      - Get the global mean temperature change for specified years and specified percentiles (Delta T). These define our scenarios.
+      - Select the 30-year period from the target model (all ensemble members) where they match the Delta T for each scenario.
+    variables:
+      tas_cmip:
+        short_name: tas
+        preprocessor: preprocessor_global
+        additional_datasets: *cmip5
+      tas_target:
+        short_name: tas
+        preprocessor: preprocessor_global
+        additional_datasets: *target
+    scripts:
+      global_matching:
+        script: kcs/global_matching.py
+        scenario_years: [2050, 2085]  # TODO: Adapt diagnostic to plot and print these points (like in figure 3a)
+        scenario_percentiles: [P10, P90]
+
+  local_resampling:
+    description: >
+      - Divide the 30-year dataset into 5-year blocks
+      - Create all possible combinations out of these 30/5 = 6 periods and x ensemble members (may contain the same block multiple times, but less is better and maximum three times (see third item below))
+      - Determine the 1000 best ...
+      - Determine the final best
+    variables:
+      pr_target:
+        short_name: pr
+        preprocessor: preprocessor_local
+        additional_datasets: *target
+      tas_target:
+        short_name: tas
+        preprocessor: preprocessor_local
+        additional_datasets: *target
+      pr_cmip:
+        short_name: pr
+        preprocessor: preprocessor_local
+        additional_datasets: *cmip5
+      tas_cmip:
+        short_name: tas
+        preprocessor: preprocessor_local
+        additional_datasets: *cmip5
+    scripts:
+      resample:
+        script: kcs/local_resampling.py
+        control_period: [1981, 2010]
+        n_samples: 8
+        scenarios:
+          ML_MOC:
+            description: "Moderate warming / low changes in seasonal temperature & precipitation"
```

Comment:
Add to description " / middle of century" to distinguish from the next scenario. Also for the other scenarios below.